cmd/manager/main.go

```diff
@@ -182,7 +182,9 @@ func init() {
 
 func execute() {
 	// update GOMAXPROCS to container cpu limit if necessary
-	_, err := maxprocs.Set(maxprocs.Logger(log.Info))
+	_, err := maxprocs.Set(maxprocs.Logger(func(s string, i ...interface{}) {
```

Comment:
Maybe also include
```
	// maxprocs needs an sprintf format string with args, but our logger needs a string with optional key value pairs,
	// so we need to do this translation
```
because it's not clear just from the signatures why we need it?

Reply:
Will add